### PR TITLE
e2e: cve scan test case

### PR DESCRIPTION
**Pull request by subhashkhileri**
## Fixes 

https://issues.redhat.com/browse/HAC-3936

## Description

- Validate CVE clair scan results in pipeline run details page.
- Click on Vulnerabilities  view logs and validate Logs.
- Validate CVE clair scan results in pipeline run List page.
- Validate Clair scan node on Pipeline runs details page.
- Validate Logs on Logs Tab in Drawer Panel.

## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): Test

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: integration-tests/support/pages/tabs/PipelinerunsTabPage.ts [hunk 1]
```diff
@@ -47,6 +47,51 @@ export class DetailsTab {
...
+    cy.contains(pipelinerunsTabPO.PF4TableRow, section)
+      .contains(value)
+      .scrollIntoView()
+      .should('be.visible');
+  }
+
+  static checkVulScanOnClairDrawer() {
+    cy.get(pipelinerunsTabPO.drawerPanel)
+      .contains(pipelinerunsTabPO.listGroup, 'Vulnerabilities scan')
+      .contains(/Critical(\d+).*High(\d+).*Medium(\d+).*Low(\d+)/g)
```
Comment (Katka92) [inline integration-tests/support/pages/tabs/PipelinerunsTabPage.ts:65]: This is used in 3 places, would it make sense to extract it to the variable? E.g. in PO file?

Comment (subhashkhileri) [reply integration-tests/support/pages/tabs/PipelinerunsTabPage.ts:65]: I don't think we should move it to PO file as its not locator instead we can store it in var in main test and then pass it as parameter.

Comment (subhashkhileri) [reply integration-tests/support/pages/tabs/PipelinerunsTabPage.ts]: Updated. moved to variable.

**Comment by subhashkhileri** [comment]
PR check passed : https://ci.int.devshift.net/job/openshift-hac-dev-pr-check/3257/

**Comment by subhashkhileri** [comment]
[test]

**Pull request merged**